lnrpc/rpc.proto

```diff
@@ -2895,6 +2895,9 @@ message Feature {
 
 message FeeReportRequest {}
 message ChannelFeeReport {
+    /// The short channel id that this fee report belongs to.
```

Comment:
Is this at the top of the message for the sake of lncli? If so, rather leave the proto in order and add an ad-hoc struct which prints the order we want in lncli?

Reply:
It’s not only for lncli, but also rest. In 0.9 we went through and removed a bunch of structs that were purely for the purpose of formatting so I’m hesitant to start down that road again

Reply:
fwiw the rationale for doing so is that `jsonpb` now automatically converts byte slices to hex, but it only operates on `proto.Message` objects. if we had custom formatting they'd either have to implement this interface or add back this custom serialization, but all of this is avoided by just reordering the protos.

Reply:
Ah I see, good to know!